2do_fe/src/api.js

```diff
@@ -2,7 +2,7 @@ import axios from "axios";
 
 const api = axios.create({
   baseURL: "http://localhost:5001",
-  // baseURL: "https://2do-307.azurewebsites.net/",
```

Comment:
Try working on azure and see if that works as well. if not then it's okay to leave it as it is for now.

Reply:
I tried azure and there was some friction so I think for now we can leave it as